oat-sa/extension-parcc-tei
# Fix/tr 1835 zoom number line

Releted https://github.com/oat-sa/kitchen-playground/blob/test/jus-nm/files/delivery/composer.json

**Preinstall**
Composer file: https://github.com/oat-sa/kitchen-playground/blob/test/jus-nm/files/delivery/composer.json
Composer 2, php 7.2

the issue was caused by an extra check that was faulty for the zoomed graph.
and one first snap point from the edge point was missed in an array of values to snap to.

**How to test**
1) Install using provided composer file (same composer as for [NMC version](https://github.com/oat-sa/extension-tao-newmeridian-pci/pull/189)).
2) in authoring check PCIs: Custom interactions -> Graphing zoom number line interaction (not a [NMC version](https://github.com/oat-sa/extension-tao-newmeridian-pci/pull/189))
3) click on any segment to see zoomed graph
4) try to click around the first 3 divisions from the left edge
5) snapping should be consistent 

Check the video from the ticket if needed

This 1 of 2 PR's
